monai/data/utils.py

```diff
@@ -285,19 +285,28 @@ def iter_patch(
     patch_size_ = get_valid_patch_size(arr.shape, patch_size)
     start_pos = ensure_tuple_size(start_pos, arr.ndim)
 
+    # set padded flag to false if pad mode is None
+    padded = True if mode else False
```

Comment:
Could you please help add some description in the doc-string of `model` arg for the `model == None` case?

Thanks.

Reply:
Added!